Sponsors.md

```diff
@@ -1,17 +1,28 @@
 # Sponsors
 
-## Main way to sponsor: OpenCollective
+## Main way to sponsor us financially: through OpenCollective
 
 The main way to sponsor DevStaff is through [our OpenCollective page](https://opencollective.com/devstaff), which helps us gather financial contributions via a transparent system. There, you can find the available contribution options such as:
 
 * One-time contribution of a custom amount
 * One-time contributions of predefined amounts to sponsor a number of next meetups
 
-As a thank you for sponsoring a number of the upcoming meetups, we will feature you in those meetups by mentioning you and your contribution.
+After contributing, you can be listed in OpenCollective's DevStaff contributors. 
+Also, as a thank you for sponsoring a number of the upcoming meetups, we will feature you in those meetups by mentioning you and your contribution.
+See the "Awareness benefits" section below for more details.
 
-### How financial contributions are used
+## Awareness benefits
 
-The financial contributions may be used by DevStaff for various purposes and activities to support the community.
+The list below describes the awareness benefits that the sponsor receives in return:
+
+* "Thank you" message by organizers (not by company) during the sponsored meetups.
+* "Thank you" mention in our social media before and/or after the sponsored meetups.
+* The sponsor may leave leaflets on a table at the entrance during the sponsored meetups.
+* We may list the sponsor may be listed in our website's [sponsors page](https://www.devstaff.gr/sponsors).
```

Comment:
I think that this should be rephrased to something like `The sponsor will be considered to be added in our website's sponsor page`

Reply:
Done!